syne_tune/optimizer/baselines.py

```diff
@@ -83,6 +83,17 @@ def __init__(self, config_space: Dict, metric: str, mode: str, **kwargs):
         )
 
 
+class REA(FIFOScheduler):
+    def __init__(self, config_space: Dict, metric: str, **kwargs):
+        from syne_tune.optimizer.schedulers.searchers.regularized_evolution import RegularizedEvolution
```

Comment:
Any reason for why we would do an internal import? (It seems that all the imports you are using are from `core`)